applications/FluidDynamicsApplication/custom_python/fluid_dynamics_python_application.cpp

```diff
@@ -62,6 +62,7 @@ PYBIND11_MODULE(KratosFluidDynamicsApplication,m)
 //        KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,C_SMAGORINSKY);
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,CHARACTERISTIC_VELOCITY);
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m,DIVERGENCE);
```

Comment:
I'd call it `VELOCITY_H1_SEMINORM`.

Reply:
are u sure of the name? why not DIVERGENCE

Reply:
This is supposed to be in the line below... Sorry.

`VELOCITY_H1SEMINORM` to `VELOCITY_H1_SEMINORM`

Reply:
Done!